tests/integration/test_canceling_ops.py

```diff
@@ -45,7 +45,7 @@
 
 
 @pytest.mark.parametrize("task_type", params)
-def test_noop_when_operation_canceled(task_type):
+def test_noop_when_operation_canceled(clean_db, task_type):
```

Comment:
I don't fully understand why this changed, but a bunch of tests couldn't find a valid session anymore, so shimming in the `clean_db` fixture gives them a session